masschallenge/impact-api
# AC-7071 Office hours reservation view and confirmation modal

### Changes introduced in: [AC-7071](https://masschallenge.atlassian.net/browse/AC-7071):
- Filter office hours by open status and by program
### How to test
- Checkout this branch on `accelerate, impact and frontend`
- On localhost
- Navigate to this mentor's office hours [page](http://localhost:8181/officehours/bos/2019/?mentor_id=59788) and schedule office hours on his behalf.
- Masquerade as this [finalist](http://localhost:8181/admin/simpleuser/user/13884/masquerade/)
- Visit http://localhost:1234/directory/people/56778
- Click the `Available Office Hours` button
- See that a modal show up
- Follow the mockups attached on this ticket and ensure that the feature works as required.
